react/Footer/Footer.less

```diff
@@ -1,24 +1,19 @@
 @import (reference) "~seek-style-guide/theme";
 @import (reference) "./Footer.common.less";
 
-@classic-ia: ~"only screen and (min-width: 1025px)";
-
 .root {
   background-color: @sk-footer;
   width: 100%;
 }
 
 .content {
-  padding: (@grid-row-height * 4) @grid-gutter-width;
+  padding: (@grid-row-height * 4) (@grid-gutter-width / 2);
```

Comment:
Using page block means we cant easily do the above 10px rule. I'm happy to change this however this was done to match what was in the header already.... Lets move over to Slack.